doc/source/whatsnew/v0.24.0.txt

```diff
@@ -182,6 +182,7 @@ Other Enhancements
 - :func:`to_timedelta` now supports iso-formated timedelta strings (:issue:`21877`)
 - :class:`Series` and :class:`DataFrame` now support :class:`Iterable` in constructor (:issue:`2193`)
 - :class:`DatetimeIndex` gained :attr:`DatetimeIndex.timetz` attribute. Returns local time with timezone information. (:issue:`21358`)
+- :class:`Resampler` now is iterable (:issue:`15314`).
```

Comment:
Add a note say what each item is, or make the analogy to group by. Could you also make a new subsection of the docs and link to it from here.

Reply:
is it looking good?
```
:class:`Resampler` now is iterable like :class:`GroupBy` (:issue:`15314`).
```
______

> Could you also make a new subsection of the docs and link to it from here.

I'm not sure what do you mean here. Could you please explain a little bit?

Reply:
A section like http://pandas-docs.github.io/pandas-docs-travis/groupby.html#iterating-through-groups, but in doc/source/timeseries.rst